hack/test-cmd.sh

```diff
@@ -74,20 +74,21 @@ OPENSHIFT_ON_PANIC=crash openshift start --master="${API_SCHEME}://${API_HOST}:$
 OS_PID=$!
 
 if [[ "${API_SCHEME}" == "https" ]]; then
-    export CURL_CA_BUNDLE="${CERT_DIR}/admin/root.crt"
+    export CURL_CA_BUNDLE="${CERT_DIR}/system:admin/root.crt"
```

Comment:
Colons in paths break windows, let's leave directories non-provider qualified for now